<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QPaint</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>691</width>
     <height>431</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>91</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Цвет</string>
   </property>
   <widget class="QPushButton" name="red_color">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>22</y>
      <width>32</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>115</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Палитра</string>
    </property>
   </widget>
   <widget class="QPushButton" name="blue_color">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>84</y>
      <width>32</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="purple_color">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>84</y>
      <width>32</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="green_color">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>53</y>
      <width>32</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="yellow_color">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>53</y>
      <width>32</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="orange_color">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>22</y>
      <width>32</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>401</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Регуляторы цветов</string>
   </property>
   <widget class="QLabel" name="Value_1">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>71</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="Value_3">
    <property name="geometry">
     <rect>
      <x>201</x>
      <y>71</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="Value_2">
    <property name="geometry">
     <rect>
      <x>126</x>
      <y>71</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setcolor">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Задать</string>
    </property>
   </widget>
   <widget class="QDial" name="G">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>23</y>
      <width>50</width>
      <height>52</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QLabel" name="G_L">
    <property name="geometry">
     <rect>
      <x>88</x>
      <y>23</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>G:</string>
    </property>
   </widget>
   <widget class="QLabel" name="R_L_2">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>24</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>R:</string>
    </property>
   </widget>
   <widget class="QDial" name="R">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>24</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QDial" name="B">
    <property name="geometry">
     <rect>
      <x>179</x>
      <y>23</y>
      <width>50</width>
      <height>52</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QLabel" name="B_L">
    <property name="geometry">
     <rect>
      <x>163</x>
      <y>23</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>B:</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>76</width>
      <height>76</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="R_L_3">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;R:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="G_L_2">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;G:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="B_L_2">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;B:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_2"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_3"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="info">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>551</y>
     <width>810</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Программа готова к работе</string>
   </property>
  </widget>
  <widget class="QPushButton" name="open">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Открыть</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_pixmap">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Очистить</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Кисть</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QDial" name="pen_dial">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>50</width>
      <height>64</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
